packages/flame_markdown/lib/flame_markdown.dart

```diff
@@ -51,7 +51,20 @@ class FlameMarkdown {
         .map(_castCheck<InlineTextNode>)
         .toList();
     final child = _groupInlineChildren(children);
+
+    final customClassName = element.attributes['class'];
+    if (customClassName != null) {
+      if (element.tag != 'span') {
+        throw Exception(
```

Comment:
we could consider support for other inline elements, for example `**foo**{.super-bold}`, which is part of markdown-it-attrs.
but I think it is unnecessary because you can just make your custom style bold or whatever you want.